jerry-core/ecma/operations/ecma-proxy-object.c

```diff
@@ -967,6 +974,16 @@ ecma_proxy_object_define_own_property (ecma_object_t *obj_p, /**< proxy object *
       ret_value = ecma_raise_type_error (ECMA_ERR_MSG ("Trap returned truish for defining non-configurable property "
                                                        "which is configurable in the target"));
     }
+    /* ES11: 9.5.6.16.c*/
+    else if ((target_desc.flags & ECMA_PROP_IS_VALUE_DEFINED)
```

Comment:
`If IsDataDescriptor(targetDesc) ` this means `target_desc.flags & (ECMA_PROP_IS_VALUE_DEFINED | ECMA_PROP_IS_WRITABLE_DEFINED) != 0`